manager-bundle/src/Command/ContaoSetupCommand.php

```diff
@@ -94,7 +120,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $this->executeCommand(array_merge($php, [$this->consolePath], $command, $commandFlags), $output);
         }
 
-        $output->writeln('<info>Done! Please open the Contao install tool or run contao:migrate on the command line to make sure the database is up-to-date.</info>');
+        $io->info('Done! Please run the contao:migrate command to make sure the database is up-to-date.');
```

Comment:
Did you change the text on purpose?
```suggestion
        $io->info('Done! Please open the Contao install tool or run contao:migrate on the command line to make sure the database is up-to-date.');
```

Reply:
In Contao 4.13, the install tool still exists, so these changes should be reverted (as @ausi suggested) IMHO.

Reply:
Ah yes, accidentally cherry picked this. It's fixed now in e1ef8826cf6c06be7ea3e46e9e0b855b69f4c1bd.